src/dmd/root/filename.d

```diff
@@ -718,27 +718,22 @@ nothrow:
         return null;
     }
 
-    /*************************************
-     * Search Path for file in a safe manner.
-     *
-     * Be wary of CWE-22: Improper Limitation of a Pathname to a Restricted Directory
-     * ('Path Traversal') attacks.
-     *      http://cwe.mitre.org/data/definitions/22.html
-     * More info:
-     *      https://www.securecoding.cert.org/confluence/display/c/FIO02-C.+Canonicalize+path+names+originating+from+tainted+sources
+    /************************************
+     * Determine if path is safe.
+     * Params:
+     *  name = path
      * Returns:
-     *      NULL    file not found
-     *      !=NULL  mem.xmalloc'd file name
+     *  true if path is safe.
      */
-    extern (C++) static const(char)* safeSearchPath(Strings* path, const(char)* name)
+    extern (C++) static bool safePath(const(char)* name) pure @nogc
```

Comment:
this should be `private extern(D)` not `extern(C++)`.

Reply:
Note that changing this will fix the CI failure.

Reply:
Done